mara_db/sqlserver.py

```diff
@@ -17,7 +17,17 @@ def sqlserver_cursor_context(db: typing.Union[str, mara_db.dbs.SQLServerDB]) ->
     assert (isinstance(db, mara_db.dbs.SQLServerDB))
 
     cursor = None
-    connection = pyodbc.connect(f"DRIVER={{{db.odbc_driver}}};SERVER={db.host};DATABASE={db.database};UID={db.user};PWD={db.password}")
+
+    server = db.host
+    if db.port: # connecting via TCP/IP port
+        server = f"{server},{db.port}"
+    # NOTE:
```

Comment:
Since it is not feasible, do we really need this comment here? what is the role of that since sqsh can not support it?

Reply:
It as ment to be a comment for a later possible implementation for `sqlcmd`, see #39 

I removed it now.